src/common/components/ModalsManager.js

```diff
@@ -6,7 +6,9 @@ import styled from "styled-components";
 // import LoginHelper from "components/modals/LoginHelper";
 import AddInstance from "../modals/AddInstance";
 import Settings from "../modals/Settings/Settings";
+import Screenshot from "../modals/Screenshot";
 import InstanceDeleteConfirmation from "../modals/InstanceDeleteConfirmation";
+import ConfirmationModal from "../modals/ConfirmationModal";
```

Comment:
No other modal has the `modal` name in it. Follow conventions. Call it `ActionConfirmation`